packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -339,32 +416,37 @@ export const ContractType = (() => {
             }
         }
 
-        return trading_events[date][underlying];
+        return trading_events[date][underlying as string];
     };
 
-    const getTradingTimes = async (date, underlying = null) => {
+    const getTradingTimes = async (
+        date: string | null,
+        underlying: string | null = null
+    ): Promise<Record<string, never> | TTimes | Record<string, TTimes>> => {
         if (!date) {
-            return [];
+            return {};
```

Comment:
@kate-deriv please revert to what was done previously in: https://github.com/kate-deriv/deriv-app/pull/17/files#diff-d91edda55bb2d178afef3d47bc4767e0a69c4aab15f3c656c40c34f1ad531341

Reply:
I have a doubt: the returned value is used as an object in packages/trader/src/Stores/Modules/Trading/Actions/start-date.ts
<img width="1080" alt="Screenshot 2023-09-08 at 1 57 43 PM" src="https://github.com/binary-com/deriv-app/assets/121025168/a08dbd90-5f72-447e-b8c3-dee2c2834ba6">
@maryia-deriv

Reply:
I can add some additional check if it's an array, but `ContractType.getTradingTimes` is used only in this file (!)
Maybe it's better to return an empty object?

Reply:
ok, we can leave it but need to ask QA to re-test this part in this case: Duration trade param.

Reply:
I'll try to add ternary